.eslintrc

```diff
@@ -6,6 +6,7 @@
        "ignoreUrls": true,
        "ignorePattern": "^\\s*var\\s.=\\s*(require\\s*\\()|(/)"
      }],
+     "camelcase": 0,
```

Comment:
Hmm, does this mean we are disabling the `camelcase` rule? Any particular reason?